panacea/web/static/js/app.js

```diff
@@ -12,17 +12,47 @@ async function submitFile() {
   hideFileForm();
   hideResults();
   try {
-    const response = await fetch(this.action, {
+    const drugs_response = await fetch(this.action, {
       method: 'POST',
       body: new FormData(this),
       credentials: 'same-origin',
       headers: new Headers({authorization: apiAccessToken})
```

Comment:
We'll probably want `'Accept': 'application/json, text/plain, */*'` headers on all of these requests